import-scripts/annotation_suite/standardize_mutation_data.py

```diff
@@ -0,0 +1,1399 @@
+        # key-value pairs in vcf_data dictionary - "FUNC" may contain important tumor or
+        # normal sample data such as reads, alleles, etc.
+        if key.startswith("FUNC"):
+            # must replace single quotes with double quotes to avoid
+            # errors when loading string as a json
+            vcf_func_data = json.loads(str(value.replace("'", '"')))
+            for elem in vcf_func_data[0].items():
+                vcf_data[elem[0].upper().strip()] = process_datum(elem[1])
+
+    vcf_data["INFO"] = parsed_vcf_info
+
+    # the VCF "FORMAT" column stores keys that correspond to values in the "TUMOR"
+    # column and "NORMAL" column (if "NORMAL" is present) and the keys are
+    # separated by colons ":"
+    parsed_vcf_format_keys = list(map(lambda v: process_datum(v), vcf_data["FORMAT"].split(":")))
+    vcf_data["FORMAT"] = parsed_vcf_format_keys
+
+    # map the corresponding values in the VCF tumor column to the appropriate "FORMAT" keys
+    # which is also delimited by colons (":")
+    tumor_vcf_format_values = list(map(lambda v: process_datum(v), vcf_data[tumor_sample_data_col].split(":")))
+    vcf_data["MAPPED_TUMOR_FORMAT_DATA"] = dict(zip(parsed_vcf_format_keys, tumor_vcf_format_values))
+
+    # repeat above to map the corresponding values in the VCF normal column (if present)
+    if matched_normal_sample_id in vcf_data.keys():
+        normal_vcf_format_values = list(map(lambda v: process_datum(v), vcf_data[matched_normal_sample_id].split(":")))
+        vcf_data["MAPPED_NORMAL_FORMAT_DATA"] = dict(zip(parsed_vcf_format_keys, normal_vcf_format_values))
+    return vcf_data
+
+
+def calculate_allele_read_depth(allele, read_depths_by_nucleotides):
+    depth = 0
+    for nuc in allele:
+        if read_depths_by_nucleotides[nuc] != "":
+            depth += int(read_depths_by_nucleotides[nuc])
+    return str(depth)
+
+def all_values_in_list(values_to_check, input_list):
+    for value in values_to_check:
+        if not value in input_list:
+            return False
+    return True
+
+def get_vcf_variant_allele_idx(tumor_sample_format_data, normal_sample_format_data):
+    """
+        Determines the variant allele index to use based on genotype information if available.
+        If genotype information is not available or a call could not be made for a sample at
+        a given locus then use a variant allele idx of 1 by default.
+
+        Allele values:
+        - 0 = reference allele (i.e., what is in the 'REF' field)
+        - 1 = the first allele listed in 'ALT'
+        - 2 = the second allele listed in 'ALT'
+    """
+
+    variant_allele_idx = []
+    # if genotype information is available and a call was made for the sample then
+    # choose the first non-REF allele seen in the sample genotype
+    if "GT" in tumor_sample_format_data.keys() and not tumor_sample_format_data["GT"] in ["", ".", "./."]:
+        tumor_sample_genotype_info = tumor_sample_format_data["GT"].split("/")
```

Comment:
The split also needs to be done for vertical bar .. I'm not sure what the distinction is between a vertical bar separator or a slash separator, but both are present in the example MAF from the reference documentation